### PR TITLE
Multiple synchronous components support

### PR DESCRIPTION
**Pull request by geofjamg**
Signed-off-by: Geoffroy Jamgotchian <geoffroy.jamgotchian@rte-france.com>

**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem ?** *If so, link to this issue using `'#XXX'` and skip the rest*
No

**What kind of change does this PR introduce?** *(Bug fix, feature, docs update, ...)*
feature

**What is the current behavior?** *(You can also link to an open issue here)*
There is no support for multiple synchronous components .

**What is the new behavior (if this is a feature change)?**
Multiple synchronous components are support, one loadflow is ran on each synchronous component.

**Does this PR introduce a breaking change or deprecate an API?** *If yes, check the following:*
- [ ] The *Breaking Change* or *Deprecated* label has been added
- [ ] The migration guide has been updated in the github wiki *(What changes might users need to make in their application due to this PR?)*

**Other information**:

(if any of the questions/checkboxes don't apply, please delete them entirely)

### CODE REVIEW

File: src/main/java/com/powsybl/openloadflow/ac/outerloop/AcloadFlowEngine.java [hunk 1]
```diff
@@ -148,11 +148,15 @@ public AcLoadFlowResult run() {
 
         LOGGER.debug(Markers.PERFORMANCE_MARKER, "Ac loadflow ran in {} ms", stopwatch.elapsed(TimeUnit.MILLISECONDS));
 
-        AcLoadFlowResult result = new AcLoadFlowResult(networks, outerLoopIterations, nrIterations, runningContext.lastNrResult.getStatus(),
+        AcLoadFlowResult result = new AcLoadFlowResult(network, outerLoopIterations, nrIterations, runningContext.lastNrResult.getStatus(),
                 runningContext.lastNrResult.getSlackBusActivePowerMismatch());
 
         LOGGER.info("Ac loadflow complete (result={})", result);
```
Comment (annetill) [inline src/main/java/com/powsybl/openloadflow/ac/outerloop/AcloadFlowEngine.java:154]: Can you please add the number of the network in the log when AC loadflow complete ?

File: src/main/java/com/powsybl/openloadflow/network/impl/LfNetworkLoaderImpl.java [hunk 2]
```diff
@@ -39,115 +41,128 @@
...
 
         for (Bus bus : buses) {
-            LfBusImpl lfBus = LfBusImpl.create(bus);
+            LfBusImpl lfBus = createBus(bus, voltageRemoteControl, loadingContext, report, voltageControllerCount, controllerBusToControlledBusId);
             lfNetwork.addBus(lfBus);
+        }
 
-            bus.visitConnectedEquipments(new DefaultTopologyVisitor() {
+        if (voltageControllerCount[0] == 0) {
+            throw new PowsyblException("Component without any voltage control");
```
[the remark this answers is not in the document]
Comment (geofjamg) [reply src/main/java/com/powsybl/openloadflow/network/impl/LfNetworkLoaderImpl.java:53]: Right, I am going to change to just skip components without at least on bus that can control voltage.

File: src/main/java/com/powsybl/openloadflow/ac/outerloop/AcloadFlowEngine.java (hunk 1 above)
Comment (geofjamg) [reply src/main/java/com/powsybl/openloadflow/ac/outerloop/AcloadFlowEngine.java:154]: Ok

File: src/main/java/com/powsybl/openloadflow/network/impl/LfNetworkLoaderImpl.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (geofjamg) [reply src/main/java/com/powsybl/openloadflow/network/impl/LfNetworkLoaderImpl.java]: Done

File: src/main/java/com/powsybl/openloadflow/ac/outerloop/AcloadFlowEngine.java (hunk 1 above)
Comment (geofjamg) [reply src/main/java/com/powsybl/openloadflow/ac/outerloop/AcloadFlowEngine.java]: Done

**Pull request merged**